client/src/site-search/index.tsx

```diff
@@ -17,14 +18,16 @@ export function SiteSearch() {
   const page = searchParams.get("page");
   React.useEffect(() => {
     if (query) {
-      let title = `Search: "${query}"`;
+      let title = `Search results for "${query}"`;
```

Comment:
I don't know if this is such a good idea. I think it was more practical before. 

BEFORE:
<img width="211" alt="Screen Shot 2021-02-23 at 9 11 58 AM" src="https://user-images.githubusercontent.com/26739/108855608-3135b380-75b7-11eb-900d-72d25e6a2c3f.png">

AFTER:
<img width="232" alt="Screen Shot 2021-02-23 at 9 12 15 AM" src="https://user-images.githubusercontent.com/26739/108855651-3bf04880-75b7-11eb-9cf9-080eb3e4fb30.png">

I made those document titles short for a reason. 
Arguably, it was weird before (on `main` branch as of now) that the `<h1>` said "Results: ..." and the `document.title` said "Search: ...". But that can easily be fixed. But the document nevertheless still needs to be a bit shorter.

Reply:
See https://github.com/mdn/yari/pull/3009#discussion_r581143005

Reply:
I get what you are saying here though but, how many times do you have so few tabs open that a single tab is actually that wide? 😄  More often than not you will hover over the tab or just switch to it because of the favicon.

Reply:
Yeah. Perhaps the easiest is to just do whatever we do with the `<h1>` to be exactly the same (minus any markup). However...

One option to consider is to inject the `hitCount` there. I.e. `13 search results for "video". Page 2`
That way, if you have 3 or more tabs open for different searches, they do appear different. For example, GitHub's PRs (and issues) use the PR title first instead of some generic word. This is really halpe
<img width="444" alt="Screen Shot 2021-02-23 at 11 46 41 AM" src="https://user-images.githubusercontent.com/26739/108876983-cd1dea00-75cc-11eb-9089-6adcee342903.png">
Imagine if they all started with "Pull request: HERE IS THE PR TITLE" then all I'd see in my tab bar is: "Pull re", "Pull re", "Pull re", etc. 

So I think it's incorrect to miss the fact that the `document.title` is very tight for space. 

I think I'm inclined towards `(0) "video" search results` or `(1,234) "web" search results (page 3)`